CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ about why a change log is important.
 
 ### Added
 - Add official support for Ruby 3.2 (#157)
+- Add support for Rails 7.1 (#166)
```

Comment:
The changelog is organized in reverse chronological order, i.e., the new change should go to the top of the list.